user/getting-started.md

```diff
@@ -107,6 +110,11 @@ time="2017-07-12T14:11:13Z" level=debug msg="starting server"
 time="2017-07-12T14:11:13Z" level=debug msg="registering gRPC service"
 time="2017-07-12T14:11:13Z" level=info msg="starting gRPC server"
 ```
+
+Notice how in this case we need to share the host Docker server's Unix socket with
```

Comment:
Well, the objective of this PR wasn't an overwrite of the "Overriding driver images section" but since I actually agree that the easiest option is not shown I've changed it.